### PR TITLE
Commands

### PR DESCRIPTION
**Pull request by tstrachota**
Tests depend on: https://github.com/theforeman/hammer-cli/pull/241

### CODE REVIEW

**Comment by tstrachota** [comment]
@ares ready for initial review. I'll probably add translating of listing mode from `whitelist/blacklist/none` to numbers.

**Comment by tstrachota** [comment]
@ares, I pushed the tests and consider the branch to be ready for review now.

File: lib/hammer_cli_foreman_virt_who_configure/config.rb [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        end
+        label _('Schedule') do
+          field :_interval, _('Interval')
+          field :last_report_at, _('Last Report At'), Fields::Date
+        end
+        label _('Connection') do
+          field :satellite_url, _('Satellite server')
+          field :hypervisor_id, _('Hypervisor ID')
+          field :_filtering_mode, _('Filtering')
+          field :blacklist, _('Filtered hosts'), Fields::Field, :hide_blank => true
```
Comment (ares) [inline lib/hammer_cli_foreman_virt_who_configure/config.rb]: I think the translations need to be swapped with whitelist which filters hosts to be parsed, while blacklist excludes hosts

File: lib/hammer_cli_foreman_virt_who_configure/config.rb [hunk 2]
```diff
@@ -0,0 +1,222 @@
...
+        end
+        conf
+      end
+
+      build_options
+    end
+
+    class FetchCommand < HammerCLIForeman::Command
+      command_name "fetch"
+      action :deploy_script
```
Comment (ares) [inline lib/hammer_cli_foreman_virt_who_configure/config.rb:115]: I guess this causes deploy and fetch script have the same help
``` 
deploy                        Renders a deploy script for the specified virt-who configuration
fetch                         Renders a deploy script for the specified virt-who configuration
```

could deploy script also say it runs the script right away?

**Comment by ares** [comment]
2 nits found during testing, otherwise looks good :+1:

**Review by ares — CHANGES_REQUESTED** [review]

**Comment by tstrachota** [comment]
@ares updated

**Comment by ares** [comment]
Ack, works nicely and covers it all :+1:, once this gets in, I think we can start moving it under theforeman organization. Probably best to ask on #theforeman-dev irc channel

**Comment by tstrachota** [comment]
Meerged as https://github.com/tstrachota/hammer-cli-foreman-virt-who-configure/commit/f23454760595b3cf37ab33d4719c418dfa1a25bb and https://github.com/tstrachota/hammer-cli-foreman-virt-who-configure/commit/04ac873b9e27a2ec94947ec9a50f16a4c41640d1

**Pull request closed without merging**